Automattic/happychat-client
# SSR Troubleshooting: Set up stats

See pbvpgB-13I-p2

Updates Tracks events from #316 to fire when the form is submitted instead of when data is loaded.

### How to test

Submit the form with a few different SSRs in the "Any more info you want to share?" section, and see which Tracks events fire.

You can find links to a few example SSRs in the Test Plan of D60710-code.
